src/app/admin/dashboard/tabele/page.tsx

```
@@ -489,6 +508,99 @@ export default function AdminTablesCleanPage() {
+                    </>
+                  ) : (
+                    <>
+                      <Upload className="w-4 h-4" /> Parse & Update
+                    </>
+                  )}
+                </button>
+                <a
+                  href="/f1/tabelepage"
+                  className="px-4 py-2 border border-gray-300 rounded-lg text-gray-700 hover:bg-gray-50"
+                  target="_blank"
+                >
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)